index.html

```diff
@@ -64,6 +64,22 @@ <h1>Patient History</h1>
 
     <div id="patient-info"></div>
     <div id="immunization-history"></div>
+    <div id="AllergyIntoleranceContent"></div>
+    <div id="MedicationStatementContent"></div>
+    <div id="MedicationRequestContent"></div>
+    <div id="MedicationContent"></div>
+    <div id="ConditionContent"></div>
+    <div id="ObservationContent"></div>
+    <div id="OrganizationContent"></div>
+    <div id="DeviceContent"></div>
+    <div id="DeviceUseStatementContent"></div>
+    <div id="DiagnosticReportContent"></div>
+    <div id="ImagingStudyContent"></div>
+    <div id="MediaContent"></div>
```

Comment:
These 2 aren't available in Epic R4 USCDI, in fact I don't see anything image or media -related. Perhaps try "Binary" or "DocumentReference"